add-ons/strimzi/README.md

```diff
@@ -0,0 +1,42 @@
+# Deploying Strimzi on OpenShift
+
+This addon provides an easy way to install an Apache Kafka cluster, using [Strimzi](http://strimzi.io/) on [Minishift](https://github.com/minishift/minishift).
+
+Prerequisites:
+
+ * [Installed and running](https://docs.okd.io/latest/minishift/getting-started/installing.html) Minishift
+ * [Installed](https://docs.okd.io/latest/minishift/command-ref/minishift_oc-env.html) OpenShift CLI
+
+## Start Minishift
+The deployment works with Minishift default resources - 2 CPUs and 2 GB RAM. Nevertheless we prefer to allocate more memory (4 GB) available to Strimzi and associated infrastructure. Start Minishift using following command to provide more RAM.
```

Comment:
```suggestion
The deployment works with Minishift default resources - 2 CPUs and 4 GB RAM.
```

Reply:
Since now minishift by default uses 4GB of RAM so no need to provide step to allocate memory if 4GB is enough.

Reply:
if there a minimum version specified? if not, we can't be certain people have the 'default'

Reply:
@gbraad ok. will add min ver